pages/_app.js

```diff
@@ -3,12 +3,12 @@ import Layout from "../components/Layout";
 
 function MyApp({Component, pageProps}) {
   return (
-    <>
+    <main>
       <GlobalStyles />
       <Layout>
```

Comment:
Please move `<GobalStyles />` up one level (outside of main). It should affect everything.

Reply:
Done. 😄